### ISSUE TITLE
Serialization of XSLT results with/without disable-output-escaping

### ISSUE THREAD

**Issue by xatapult** [xproc/Vnext#42]
In the result of an XSLT stylesheet you can turn `disable-output-escaping` on (on `<xsl:value-of.../>`). Turning it on makes for instance `>` and `<` characters *not* to be serialized as `&gt;` and `&lt;` but as the characters themselves. This is useful if you have for instance a piece of XML in a string variable that you want serialized as XML, not as text.

However, serializing an XSLT result that uses this feature in an XProc pipeline does not retain this. Maybe we should do something about it.

**Comment by ndw** [comment]
I don't think `disable-output-escaping` is the way forward. You should be able to do it more cleanly with character maps at serialization time.

**Comment by xatapult** [comment]
Sure, but what if some text nodes have `disable-output-escaping` on and some have it off? Character maps are global...

**Comment by ndw** [comment]
Yeah. You have to manage things a little more carefully. But `disable-output-escaping` can produce data models that can't be serialized. It exists now only for backwards compatibility reasons, processors aren't required to support it.
